src/helpers.coffee

```diff
@@ -31,13 +32,16 @@ jade = async ({source, target, data}) ->
   render = _jade.compile source.content, filename: source.path
   target.content = render data
 
+handlebars = async ({source, target, data}) ->
+  source.content ?= yield read source.path
+  target.content = _handlebars source.content, data
```

Comment:
Not a promise, no, but I don't think the others are making uses of promises either. `panda-template` does give you the render function though, so I didn't need the extra line to define `render` like jade and the other compilers are doing. Here's what `panda-template` exports:
```coffee
module.exports = (template, context) ->
  (H.compile template, noEscape: true)(context)
```
Where `H` is handlebars with `swag` and some custom panda helpers registered.